xbmc/Autorun.cpp

```diff
@@ -85,8 +85,14 @@ bool CAutorun::ExecuteAutorun(const std::string& path)
   bool success = false;
 
 #ifdef HAS_CDDA_RIPPER
-  if (CServiceBroker::GetSettingsComponent()->GetSettings()->GetInt(CSettings::SETTING_AUDIOCDS_AUTOACTION) == AUTOCD_RIP &&
-      pInfo->IsAudio(1) && !CServiceBroker::GetSettingsComponent()->GetProfileManager()->GetCurrentProfile().musicLocked())
+  const auto& settings = CServiceBroker::GetSettingsComponent()->GetSettings();
+  const AutoCDAction action =
+      static_cast<AutoCDAction>(settings->GetInt(CSettings::SETTING_AUDIOCDS_AUTOACTION));
+  if (action == AutoCDAction::RIP && pInfo->IsAudio(1) &&
+      !CServiceBroker::GetSettingsComponent()
+           ->GetProfileManager()
+           ->GetCurrentProfile()
+           .musicLocked())
```

Comment:
How about

```c++
  const auto& settings = CServiceBroker::GetSettingsComponent()->GetSettings();
  const auto& profileManager = CServiceBroker::GetSettingsComponent()->GetProfileManager();
```

```c++
  const AutoCDAction action =
      static_cast<AutoCDAction>(settings->GetInt(CSettings::SETTING_AUDIOCDS_AUTOACTION));
  if (action == AutoCDAction::RIP && pInfo->IsAudio(1) &&
      !profileManager->GetCurrentProfile().musicLocked())
```

Reply:
sure. ultimately i don't put any effort into cosmetics since clang-format will find some way to mangle it in any case..

Reply:
Even without these minor improvements your work is making a big difference in quality. Thanks!